index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 /* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(fn => {
```

Comment:
Ok, I think the `xo` is trying to throw out errors if you run the test.
maybe you can turn off above rules so that people won't get this error?